_drafts/newsletter-001.md

```diff
@@ -279,8 +383,12 @@ Special section for other news (up to 15) in a one-liner format:
 - optional small image.
 -->
 
-- Vlad Zhukov shared on Twitter videos of [his space game prototype](https://twitter.com/VladZhukov0/status/1157636331629137921)
-  and [his experiments with Voronoi diagram for procedural destruction](https://twitter.com/VladZhukov0/status/1162462543530643457).
+- [Vlad Zhukov](https://twitter.com/VladZhukov0) shared
+  [his first Youtube devlog](https://youtu.be/7NojrtICE1k)
```

Comment:
Sorry for bringing this up... I think we need to use "theirs" instead of "his" and "her" everywhere. The reason being: even though his/her may be correct (assuming we ask the authors about their correct pronounces), the exact difference is not relevant to the topic of the blog. All we want to say is that this is a creative person, and "their" makes this simple to express (and always safe to say).

Reply:
Done :)